meetings/2023-03-29.md

```diff
@@ -38,41 +41,145 @@
 
-
 ### Quote and cite content
 URL: https://github.com/solid/solid-wg-charter/pull/4
+* SC: Editorial change. Multiple paragraphs are copied/pasted from the Protocol directly into the Charter but lacks proper citing. Proposal is to update that. Suggestions/objections? 
+* PAC: I thought this was merged. Very minor thing but stylesheet as it is today makes quotes very prominent (blue background). We could change. Otherwise good to cite. The way the citations are presented now is a bit too disruptive. Even the quotes break stream of the text. Maybe there's a less disruptive way of putting this. Otherwise, no objections to this PR. 
+* SC: Noticed the background color. I can update the PR. 
+* PAC: One quote has been inserted in the PR, but that's from a different PR. 
+* SC: There were two PRs. The other one got merged. 
+
+ACTION: SC to update stylesheet and merge.
 
 
 ### Clarify and reference Solid Protocol, CG, use cases
 URL: https://github.com/solid/solid-wg-charter/pull/8
+* SC: We have one approval (PAC). This is just changing Solid specification to Protocol, and the other one is reference to Solid Community Group, and the other one where it says Pods I think it means to say server. Fourth is repository with use cases and user stories. We want to put there as much as possible of what the group has done. 
+* VB: Approved.
+* eP: Approved.
 
+ACTION: Merge.
 
 ### Fix success criteria (testing, interop, horizontal review)
 URL: https://github.com/solid/solid-wg-charter/pull/10
-
+* SC: This is trying to align content with the latest Charter template W3C is using. This PR links to an issue mentioning that. And then responding/filling in content based on that. Key areas: testing and interop, horizontal reviews. Main additions are clarifying what will count as interoperable (can be verified with open test suites) and committing to the fact we're willing to work on tests/have a testing plan as early as possible and how to promote interoperability in testing. We have work in the CG around testing, QA. Some of the specs have already been moving in that direction and we already have test suites that are testing that. Building on the W3C QA activities which goes back two decades of work. This is important for any group to show properly how interoperability can be verified and how implementation reports are obtained, etc. The last part is not too different from previous template. 
+* PAC: I see a sentence you did not copy from the template: 
+  https://w3c.github.io/charter-drafts/charter-template.html#success-criteria (second to last sentence of this section)
+* PAC: Was wondering if it was deliberate to omit the sentence, now I see it was not. Since it's part of the template we might as well include.
+* SC: I noticed in the source code, when you clone this Charter template and look at it I see a different version from the one on the GitHub pages one. Which may explain why we missed each other there (in this PR's comments)
```

Comment:
```suggestion
* SC: This is trying to align content with the latest Charter template W3C is using. This PR links to an issue mentioning that. And then responding/filling in content based on that. Key areas: testing and interop, horizontal reviews. Main additions are clarifying what will count as interoperable (can be verified with open test suites) and committing to the fact we're willing to work on tests/have a testing plan as early as possible, and how to promote interoperability in testing. We have work in the CG around testing, QA. Some of the specs have already been moving in that direction, and we already have test suites that are testing that. Building on the W3C QA activities which go back over two decades of work. This is important for any group to properly show how interoperability can be verified and how implementation reports are obtained, etc. The last part is not too different from previous template.
* PAC: I see a sentence you did not copy from the template (second to last sentence of [this section](https://w3c.github.io/charter-drafts/charter-template.html#success-criteria))
* PAC: I was wondering if it was deliberate to omit the sentence; now I see it was not. Since it's part of the template, we might as well include it.
* SC: I noticed in the source code, when you clone this Charter template and look at it, I see a different version than is on GitHub pages. Which may explain why we missed each other there (in this PR's comments).
```